lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3224,8 +3224,8 @@ prop_balanceTransactionValid
                         ]
             Left (ErrBalanceTxAssetsInsufficient err) -> do
                 let missing = view #shortfall err
-                let missingCoin = view #coin missing == Coin 0
-                let missingTokens = view #tokens missing == mempty
+                let missingCoin = Value.coin missing == mempty
+                let missingTokens = Value.isAdaOnly missing
```

Comment:
Hm... doesn't it look like the original check (and new) is the opposite of what it should be?

Reply:
> Hm... doesn't it look like the original check (and new) is the opposite of what it should be?

Good point! I'll try to create another PR that improves the naming a bit.